openhands/core/config/mcp_config.py

```diff
@@ -141,22 +145,45 @@ def from_toml_section(cls, data: dict) -> dict[str, 'MCPConfig']:
         return mcp_mapping
 
 
+def add_search_engine(app_config: "AppConfig") -> MCPStdioServerConfig | None:
```

Comment:
NIT: lets also add this as a class method to `OpenHandsMCPConfig` so that we can override it in saas and return `None` (just to be double sure that it returns `None` in saas)

Reply:
done!